ci-operator/config/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-main.yaml

```diff
@@ -438,6 +438,19 @@ tests:
     cluster_profile: azure4
     workflow: openshift-e2e-cert-rotation-short-azure
   timeout: 8h0m0s
+- as: e2e-tests-extension
+  steps:
+    test:
+    - as: tests-extension
+      commands: |
+        echo "Build binary kube-apiserver-tests-ext"
+        make tests-ext-build
+        echo "Running ./kube-apiserver-tests-ext with sanity test"
+        ./kube-apiserver-tests-ext run-test -n "[Jira:KAS][sig-api-machinery] sanity test should always pass [Suite:openshift/cluster-kube-apiserver-operator/conformance/parallel]"
```

Comment:
Are you sure we need this? iiuc `origin` would discover it and run automatically?

Reply:
With this interface, we can run some other extension tests. such as we see https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/67975/rehearse-67975-pull-ci-openshift-cluster-kube-apiserver-operator-release-4.20-e2e-tests-extension/1954830351985545216/artifacts/e2e-tests-extension/tests-extension/build-log.txt

Reply:
We plan to port some cases from https://github.com/openshift/openshift-tests-private/blob/master/test/extended/apiserverauth/apiserver.go, also we can add some related cases to this.